src/App.js

```diff
@@ -14,11 +15,91 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: ['Jam'],
+  }
+
+  selectedGoods = () => this.state.selectedGoods.join(' ');
```

Comment:
you have variable with the same name in the state, method should have verb in its name